adafruit_hid/consumer_control.py

```diff
@@ -52,6 +53,14 @@ def __init__(self):
         # View bytes as a single 16-bit number.
         self.usage_id = memoryview(self.report)[0:2]
 
+        # Do a no-op to test if HID device is ready.
+        # If not, wait a bit and try once more.
+        try:
+            self.send(0x0)
+        except OSError:
+            time.sleep(1)
+            self.send(0x0)
```

Comment:
The second `self.send()` will throw `OSError` if it fails, just like the first. But that won't get caught so it will bubble up to the user.

Reply:
I'm not sure the error number will be the same across all ports, or even from 2.x to 3.0. There are a couple of different conditions that are reported up that might be recoverable.

Reply:
I was afraid we couldn't narrow down on the error number; cest la vie. For the exception, I was just wondering if there was a better way to report than the status LED. But, since we're dealing with situations very early in the program running, REPL output is unlikely. And clearly, if the REPL is running, HID is probably fully initialized. Only other way around it is to have the user put their instance creation in a `try` and catch any exception themselves...

This looks good to me as is then. I'll let @tannewt review and final approve.